src/components/common/DocumentsDelivery.vue

```diff
@@ -20,7 +20,7 @@
 
         <v-row v-if="isRoleStaff" no-gutters class="mt-6">
           <v-col cols="3" class="px-0">
-            <label :class="{ 'error-text': documentDeliveryInvalid }"><strong>User Account</strong></label>
+            <label :class="{ 'error-text': documentDeliveryInvalid }"><strong>Completing Party</strong></label>
```

Comment:
While a staff can access the filing, a registered SP/GP user can also access it. I think we need a conditional rendering for this component. Furthermore, I noticed "User Account" appears in two places in this file, and the other one is in line 43. Can you verify with the UI/UX team whether either or both of them needs the change?

Reply:
Yes the "User Account" on line 43 is what renders the SP/GP view of the Document delivery component. The ticket said only change for the Completing party. However, I will verify with the UI/UX team. That would be Roy or Yui right?

Reply:
Yes, Roy and Yui are in the UI/UX team.

Reply:
Looks like you were right Lewis, thanks for that awesome catch!